src/app/containers/FrontPage/index.jsx

```diff
@@ -28,6 +29,7 @@ const EnhancedFrontPageContainer = compose(
   withLoading,
   withError,
   withData,
+  withVariant,
```

Comment:
could this be done higher up in the stack? as right now, this will check them loading state, check for errors, validate the data and then potentially redirect.